.github/workflows/tests.yml

```
@@ -197,11 +197,11 @@ jobs:
 
       - name: Core System Tests
         if: matrix.build_type == 'system' && matrix.target == 'core'
-        run: bin/rspec spec/system --format documentation
+        run: PARALLEL_TEST_PROCESSORS=4 bin/turbo_rspec spec/system --format documentation
```

```suggestion
        run: PARALLEL_TEST_PROCESSORS=4 bin/turbo_rspec spec/system
```
Don't think documentation formatting will work with parallel tests (and even if it did, it would be horrible to read)